screenwriter/screenwriter.go

```diff
@@ -0,0 +1,60 @@
+package screenwriter
+
+import (
+	"fmt"
+	"io"
+	"strings"
+)
+
+type ScreenWriter struct {
+	writer   io.StringWriter
+	previous string
+}
+
+func New(writer io.StringWriter) *ScreenWriter {
+	return &ScreenWriter{
+		writer: writer,
+	}
+}
+
+func (w *ScreenWriter) ClearPrevious() error {
```

Comment:
Should this be private? i.e. `clearPrevious`?

Reply:
It could potentially be useful outside of this package. However as we're not actually using it in the `main` package, it might not be. I've made it private.

Reply:
I think the main this is that the commit message said that the only method was the `Display` method.